ext/rbs_extension/location.c

```diff
@@ -181,7 +184,7 @@ static VALUE location_add_required_child(VALUE self, VALUE name, VALUE start, VA
   rg.start = rbs_loc_position(FIX2INT(start));
   rg.end = rbs_loc_position(FIX2INT(end));
 
-  rbs_loc_add_required_child(loc, rbs_find_constant_id_from_ruby_symbol(name), rg);
```

Comment:
This was a mistake in the "dummy" Ruby-based implementation. The Ruby symbols were already interned by the fact they were used in Ruby code, so the lookup here always succeeded.

We need an "insert" operation here, not a "find", to ensure that if a child is added with a new name, that its name is present in the constant pool.

Reply:
Since it's technically a fix, can this and other similar changes be upstreamed separately?

Reply:
It wasn't an issue in the previous implementation, and it only matters starting in this PR. It's small enough that I don't think it's worth shipping separately.